wled00/usermods_list.cpp

```diff
@@ -173,6 +173,10 @@
   #include "../usermods/boblight/boblight.h"
 #endif
 
+#ifdef USERMOD_INTERNAL_TEMPERATURE
+  #include "../usermods/Internal_Temperature_v2/usermod_internal_temperature.h"
```

Comment:
Please agree with @lost-hope on a folder and name for this usermod (in MM fork its usermods/mcu_temp). It might be a minor point for this PR to "Aircoookie" WLED, but we should do our best to avoid having two usermods with almost identical code but different name and folder.

Reply:
I don't want to waste @lost-hope's time. After we merge this PR I will create new one in the MM fork and will replace usermods/mcu_temp with this usermod.